### ISSUE TITLE
Map window "bounce" when zooming in

### ISSUE THREAD

**Issue by talllguy** [mpetroff/print-maps#3]
[Here is a screencast](http://youtu.be/Mab7tSgQkxs) of an odd behavior when zooming in.

Chrome Version 40.0.2214.111 m
Windows 7 x64

**Comment by talllguy** [comment]
Note that while zooming with the scrollwheel, the behavior is further exaggerated.

**Comment by mpetroff** [comment]
Do you have the same behavior with [Mapbox's examples](https://www.mapbox.com/mapbox-gl-js/examples/)?

**Comment by talllguy** [comment]
@mpetroff No, not on the mapbox examples. Add oddly not on this machine on your site either, similar setup: Chrome latest Win8.1x64. I wonder if this and #2 were caused by some firewall issue at my office.

**Comment by mpetroff** [comment]
Chrome tried loading the file in #2 for me as well, so that definitely wasn't your firewall. I very much doubt this has to do with the network either.
